dts/bindings/sram/mmio-sram.yaml

```diff
@@ -0,0 +1,33 @@
+#
+# Copyright (c) 2018, Linaro Limited
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
+title: Generic on-chip SRAM
+version: 0.1
+
+description: >
+    This binding gives a generic on-chip SRAM description
+
+properties:
+    compatible:
+      type: string
+      category: required
+      description: compatible strings
+      constraint: "mmio-sram"
+      generation: define
+
+    reg:
+      type: array
+      description: mmio register space
+      generation: define
+      category: required
+
+    label:
+      type: string
+      category: optional
+      description: Human readable string describing the device (used by Zephyr for API name)
+      generation: define
+
```

Comment:
I can see property 'device_type' on some mmio-nodes (dts/x86/apollo_lake.dtsi).
Should it be taken into account ?
Otherwise, sounds good

Reply:
We should remove device_type = 'memory' from the nodes.  You good w/handling that in a separate PR.

Reply:
Sure